tanner/web/server.py

```diff
@@ -0,0 +1,98 @@
+        snare_uuid = request.match_info['snare_uuid']
+        return{
+        'snare' : snare_uuid
+        }
+
+    @aiohttp_jinja2.template('snare-stats.html')
+    async def handle_snare_stats(self, request):
+        snare_uuid = request.match_info['snare_uuid']
+        snare_stats = await self.api.return_snare_stats(snare_uuid)
+        return {
+        'snare_stats' : snare_stats
+        }
+
+    @aiohttp_jinja2.template('sessions.html')
+    async def handle_sessions(self, request):
+        snare_uuid = request.match_info['snare_uuid']
+        params = request.url.query
+        applied_filters = {'snare_uuid': snare_uuid}
+        try:
+            if 'filters' in params:
+                applied_filters = {filt.split(':')[0] : filt.split(':')[1] for filt in params['filters'].split()}
+                if 'start_time' in applied_filters:
+                    applied_filters['start_time'] = float(applied_filters['start_time'])
+                if 'end_time' in applied_filters:
+                    applied_filters['end_time'] = float(applied_filters['end_time'])
+        except Exception as e:
+            self.logger.error('Filter error : %s' % e)
+            result = 'Invalid filter definition'
+        else:
+            sess_uuids = await self.api.return_sessions(applied_filters)
+            sessions = []
+            for sess_uuid in sess_uuids:
+                sess = await self.api.return_session_info(sess_uuid)
+                sessions.append(sess)
+        return {
+        'sessions' : sessions
+        }
+
+    @aiohttp_jinja2.template('session.html')
+    async def handle_session_info(self, request):
+        sess_uuid = request.match_info['sess_uuid']
+        session = await self.api.return_session_info(sess_uuid)
+        return {
+        'session' : session
+        }
+
+    def setup_routes(self, app):
+        app.router.add_get('/', self.handle_index)
+        app.router.add_get('/snares', self.handle_snares)
+        app.router.add_resource('/snare/{snare_uuid}').add_route('GET', self.handle_snare)
+        app.router.add_resource('/snare-stats/{snare_uuid}').add_route('GET', self.handle_snare_stats)
+        app.router.add_resource('/session/{sess_uuid}').add_route('GET', self.handle_session_info)
+        app.router.add_resource('/{snare_uuid}/sessions').add_route('GET', self.handle_sessions)
+        app.router.add_static('/static/', path='./static')
+
+    def create_app(self, loop):
+        app = web.Application(loop= loop)
+        aiohttp_jinja2.setup(app,
+            loader= jinja2.FileSystemLoader('templates'))
+        self.setup_routes(app)
+        return app
+
+    def start(self):
+        loop = asyncio.get_event_loop()
+        self.redis_client = loop.run_until_complete(redis_client.RedisClient.get_redis_client())
+        self.api = api.Api(self.redis_client)
+        app = self.create_app(loop)
+        web.run_app(app, host= '0.0.0.0', port= 8091)
```

Comment:
I think it'll run localhost only unless we make it public as it has different port than tanner.